docs/guide/api-environment-runtimes.md

```diff
@@ -82,25 +82,22 @@ A Vite Module Runner allows running any code by processing it with Vite plugins
 One of the goals of this feature is to provide a customizable API to process and run code. Users can create new environment factories using the exposed primitives.
 
 ```ts
-import { DevEnvironment, RemoteEnvironmentTransport } from 'vite'
+import { DevEnvironment, HotChannel } from 'vite'
 
 function createWorkerdDevEnvironment(name: string, config: ResolvedConfig, context: DevEnvironmentContext) {
-  const hot = /* ... */
   const connection = /* ... */
-  const transport = new RemoteEnvironmentTransport({
+  const transport: HotChannel = {
+    on: (listener) => { connection.on('message', listener) },
     send: (data) => connection.send(data),
-    onMessage: (listener) => connection.on('message', listener),
-  })
+  }
 
   const workerdDevEnvironment = new DevEnvironment(name, config, {
     options: {
       resolve: { conditions: ['custom'] },
       ...context.options,
     },
-    hot,
-    remoteRunner: {
-      transport,
-    },
+    hot: true,
+    transport,
```

Comment:
Maybe this should be `channel` or `hotChannel`? `transport` is also used on the module runner side (ModuleRunnerTransport).

Reply:
Yeah, `transport: HotChannel` typing feels confusing. I liked the symmetry of  `DevEnvironmentContext.transport` and `ModuleRunnerOptions.transport` though, so if we change this on environment side, then we might want to align on runner side.

Reply:
I don't think naming stuff a blocker and we can follow up later if needed.